bookmarket-backend/src/main/java/edu/oakland/sophomoreproject/dependencies/sqlite/listings/ListingsTableAccessor.java

```diff
@@ -72,6 +69,22 @@ public Listing getListingById(int listingId) {
 	/// this can be done by doing `INSERT INTO listings ... RETURNING listing_id`
 	/// and parsing the `listing_id` column it returns with `results.getString("listing_id")`
 	public void createListing(ListingWithoutId listingWithoutId) {
-		// TODO
+		String sql ="INSERT INTO listings (title, author, description, classSubject, price, condition, createdAt, saleAvailability, sellerId" + "VALUES (?, ?, ?, ?, ?, ?, ?, ?, ?)";
+
+		Connection connection = getDatabaseConnection();
+		PreparedStatement sqlQuery = connection.prepareStatement(sql);
+
+		sqlQuery.setString(1, title);
+		sqlQuery.setString(2, author);
+		sqlQuery.setString(3, classSubject);
+		sqlQuery.setString(4,description);
+		sqlQuery.setFloat(5, price);
+		sqlQuery.setString(6, condition);
+		sqlQuery.setTimestamp(7, Timestamp.from(Instant.now()));
+		sqlQuery.setString(8, saleAvailability);
+		sqlQuery.setInt(9, sellerId);
+
+		sqlQuery.executeUpdate();
```

Comment:
I believe this should be `execute` or `executeQuery` (I forget which one it's called), but `executeUpdate` is used specifically for `UPDATE` SQL statements, but we're doing `INSERT` so we shouldn't use `executeUpdate` here

Reply:
Gotcha thank you, I wasn't entirely sure so I clicked the first one IntelliJ suggested.